video_store_management.py

```diff
@@ -0,0 +1,77 @@
+import requests
+from requests.api import post
+
+BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
+
+class VideoStoreManagement:#wrapper class allows me to interact with the API through the url. Handles all interactions with API. 
+    def __init__(self, url=BACKUP_URL):
+        self.url = url
+        
+    def create_customer(self,name,postal_code,phone):
+        query_params = {
+            "name": name,
+            "postal_code": postal_code,
+            "phone": phone
+            }
+        response = requests.post(self.url+"/customers",json=query_params)
+        print(response.json())
+        return response.json()
+    
+    def create_video(self,title, release_date):
+        query_params = {
+            "title": title,
+            "release_date": release_date,
+            "total_inventory": 1
+            
+            }
+        response = requests.post(self.url+"/videos",json=query_params)
+        print(response.json())
+        return response.json()
```

Comment:
One opportunity for improvement for many people for this project is handling error codes from the API requests. In general, when writing code to call an API, it's appropriate to check whether the request was successful or it triggered some sort of error. This is where HTTP response codes come in handy! We can usually assume that a 2XX response code means there was no error and any other response code means there was some sort of error. It's very common to have an if...elif...else chain to handle different ranges of response codes.

This applies to all the other API calls as well.